cmd/erasure-sets.go

```diff
@@ -981,7 +981,7 @@ func (s *erasureSets) startMergeWalksVersionsN(ctx context.Context, bucket, pref
 				// Disk can be offline
 				continue
 			}
-			entryCh, err := disk.WalkVersions(bucket, prefix, marker, recursive, endWalkCh)
+			entryCh, err := disk.WalkVersions(GlobalContext, bucket, prefix, marker, recursive, endWalkCh)
```

Comment:
why not 'ctx' here ?

Reply:
@vadmeste The walks live beyond the request since it will collect information for continuations of the ListObject calls.

If we use the provided context the listing will stop once the first request returns and we will see similar results as the failures above.

So this is defensive coding for now. Me may get better abort functionality as time moves on.